library/Reporting/Actions/SendMail.php

```diff
@@ -28,7 +28,8 @@ public function execute(Report $report, array $config)
 
         $mail = new Mail();
 
-        $mail->setFrom(Config::module('reporting')->get('mail', 'from', 'reporting@icinga'));
```

Comment:
Please format the code in this way:

```
$mail->setFrom(
    Config::module('reporting', 'config', true)->get('mail', 'from', 'reporting@icinga'));
);
```

Otherwise, at first glance it looks like the `get()` call belongs to `$mail`.